app/src/main/java/com/infomaniak/mail/data/models/thread/Thread.kt

```diff
@@ -90,6 +93,18 @@ class Thread : RealmObject {
     var isLocallyMovedOut: Boolean = false
     @Transient
     var numberOfScheduledDrafts: Int = 0
+    @Transient
+    private var _snoozeState: String? = null
+    @Transient
+    var snoozeEndDate: RealmInstant? = null
+    @Transient
+    var snoozeAction: String? = null
+    //endregion
+
+    //region UI data (Ignore)
+    @Ignore
+    var snoozeState: SnoozeState? by apiEnum(::_snoozeState)
+        private set
     //endregion
```

Comment:
```suggestion
    @Ignore
    var snoozeState: SnoozeState? by apiEnum(::_snoozeState)
        private set
```

Reply:
The _"UI data"_ region is supposed to be _"@.Transient & @.Ignore"_.
Here, it's different, so I removed the region.

Does it even need to be @.Ignore here ? 🤔